src/Forms/Android/ArcGISRuntime.Xamarin.Forms.Android.csproj

```diff
@@ -98,6 +98,9 @@
     <Reference Include="Validation, Version=2.4.0.0, Culture=neutral, PublicKeyToken=2fc06f0d701809a7, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Validation.2.4.15\lib\netstandard1.3\Validation.dll</HintPath>
     </Reference>
+    <Reference Include="Windows.Foundation.UniversalApiContract">
```

Comment:
It looks like this shouldn't have been included... I don't know if its necessary or will have side effects.